packages/destination-actions/src/destinations/salesforce/lead/generated-types.ts

```diff
@@ -2,49 +2,53 @@
 
 export interface Payload {
   /**
-   * Operation
+   * The Salesforce operation performed. The operations available create, update or upsert Lead records in Salesforce.
    */
   operation: string
   /**
-   * Traits
+   * The fields used to find Salesforce Lead records for updates. This is required if the Operation is Update or Upsert.
+   *
+   *   Any field can be matched on, including Record ID, External IDs, standard fields and custom fields. On the left-hand side, input the Salesforce field name. On the right-hand side, map the Segment field that contains the value.
```

Comment:
```suggestion
   *   Any field can function as a matcher, including Record ID, External IDs, standard fields and custom fields. On the left-hand side, input the Salesforce field name. On the right-hand side, map the Segment field that contains the value.
```

@kdaswani Not sure if "Any field can function as a matcher..." makes sense here. Could leave as is.

Reply:
@brennan I like it since the field is called Record Matcher. I'll add this suggestion to the sf-properties file. Thanks!